include/cereal/archives/xml.hpp

```diff
@@ -647,7 +647,8 @@ namespace cereal
       {
         try
         {
-          value = std::stold( itsNodes.top().node->value() );
+          auto wstring = std::wstring{(wchar_t *)itsNodes.top().node->value()};
```

Comment:
use an explicit cast -- static_cast or reinterpret_cast rather than c-style casts. (CI recommends reinterpret_cast)

Reply:
Done in 86e2a2bf.